src/models/stores/workspace.ts

```diff
@@ -34,9 +34,9 @@ export const WorkspaceModel = types
         Logger.logDocumentEvent(LogEventName.VIEW_SHOW_DOCUMENT, document);
       }
     };
-    const setComparisonDocument = (document?: DocumentModelType) => {
+    const setComparisonDocument = (document?: DocumentModelType | GroupVirtualDocument) => {
       self.comparisonDocumentKey = document && document.key;
-      if (document) {
+      if (document && !(document instanceof GroupVirtualDocument)) {
```

Comment:
```suggestion
      if (document && (document instanceof DocumentModelType)) {
```
Positive test seems clearer than the negative test and (slightly) reduces our dependency on `GroupVirtualDocument`.

Reply:
Uh, this got ugly quick.  See this slack thread: 

https://concord-consortium.slack.com/archives/C0303QUNU/p1571240770044500

Reply:
I am going to keep this sort of ugly check in place for now.